src/step_definitions/elements.js

```diff
@@ -304,7 +304,100 @@ defineSupportCode(function({ When, Then }) {
       .then(function(matcherResult) {
         return expect(matcherResult).to.not.be.true;
       })
-      .catch(() => Promise.resolve());
+      .catch(error => {
+        if (error.message === 'expected true to be false') {
```

Comment:
We're reaping the code in every `catch` statement in those steps.
We could add a method which handles this case and just reuse it here.